lib/couchdb.js

```diff
@@ -358,10 +358,16 @@ CouchDB.prototype.buildSort = function(mo, model, order) {
 
     if (n === idName) n = '_id';
 
-    if (m && m[1] === 'DE') obj[n] = 'desc';
-    else obj[n] = 'asc';
+    if (m && m[1] === 'DE') obj[n] = direction = 'desc';
+    else obj[n] = direction = 'asc';
     sort.push(obj);
   }
+  const skipModelView = (sort.length === 1) && ((sort[0]).hasOwnProperty('_id'));
```

Comment:
don't inject the model view property when only order by `_id`, because couchdb has its own specific index for `_id`, adding one more field results in couchdb cannot find a proper index.